.gitignore

```diff
@@ -14,6 +14,7 @@ target
 # Editors
 /.idea/
 .DS_Store
+.vscode/
```

Comment:
While I agree with this change, shouldn't do this in a feature PR.

You can locally in your own git config ignore files or even changes to files instead.